dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/logger/DatadogLogHandlerTest.kt

```diff
@@ -214,4 +229,86 @@ internal class DatadogLogHandlerTest {
                 .hasTags(emptyList())
         }
     }
+
+    @Test
+    fun `it will add the span id and trace id if we active an active tracer`(forge: Forge) {
+        // given
+        val config = DatadogConfig.Builder(forge.anAlphabeticalString()).build()
+        Datadog.initialize(mockContext(), config)
+        val tracer = Tracer.Builder().build()
+        val span = tracer.buildSpan(forge.anAlphabeticalString()).start()
+        tracer.activateSpan(span)
```

Comment:
What will happen if we don't activate Span? I think we'll end up with `dd.trace_id` set but `dd.span_id` unset for the serialized log. Is this valid scenario?

Reply:
If you check the way the trace_id and span_id are generated in the TracerExtensions ...there's no way to have one without the other. If there's no active span the trace_id and span_id will be null. Everything is taken from the activeSpan. I followed exactly what they were doing in the Java API.